packages/amplify_datastore/example/integration_test/save_test.dart

```diff
@@ -27,15 +27,40 @@ void main() {
     setUp(() async {
       await configureDataStore();
       // clear data before each test
-      await Amplify.DataStore.clear();
+      await clearDataStore();
     });
 
-    testWidgets('should save data locally', (WidgetTester tester) async {
+    testWidgets('should save a new model', (WidgetTester tester) async {
       Blog testBlog = Blog(name: 'test blog');
       await Amplify.DataStore.save(testBlog);
       var blogs = await Amplify.DataStore.query(Blog.classType);
       expect(blogs.length, 1);
       expect(blogs.contains(testBlog), isTrue);
```

Comment:
It is local. I removed 'locally' from the description since all of the tests right now run locally. Unless I added 'locally' to all the tests, it felt like I was implying that the other tests in the file are not local as well.

When we eventually have cloud sync tests, we could probably group all the local tests under one 'local' group.